build.sbt

```diff
@@ -148,7 +148,7 @@ lazy val V = new {
   val scalameta = "4.2.5"
   val semanticdb = scalameta
   val bsp = "2.0.0-M4+10-61e61e87"
-  val bloop = "1.3.5"
+  val bloop = "1.3.4+251-1d81dfe5"
```

Comment:
1.3.5 was a failed release, there's no longer a v1.3.5 tag in the Bloop repo. The new version was released last night https://scalacenter.github.io/bloop/setup

Reply:
It would be nice to reenable windows DAP tests, but I can do it later.

Reply:
I re-enabled the DAP tests on Windows, let's see how they do!